ouroboros-network-framework/src/Ouroboros/Network/Mux.hs

```diff
@@ -38,19 +38,19 @@ import           Ouroboros.Network.Driver
 
 -- |  Like 'MuxApplication' but using a more polymorphic 'Channel' type.
 --
-data OuroborosApplication (appType :: AppType) peerid ptcl m bytes a b where
+data OuroborosApplication (appType :: AppType) ptcl bytes m a b where
```

Comment:
This has a potential of simplifying clients.  The `cardano-explorer` does not need to know about `peer` at all at this level.